Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "near-abi"
+version = "0.1.0"
+edition = "2021"
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/near/near-abi-rs"
+description = "NEAR smart contract ABI primitives"
+
+[dependencies]
+borsh = { version = "0.9", features = ["const-generics"] }
+serde = { version = "1", features = ["derive"] }
+schemars = "0.8.8"
```

Comment:
Noting the unfortunate circumstance that if someone didn't want any JSON schemas in their ABI, this dependency still gets pulled in to every project (including near-sdk if the abi feature is enabled). Nothing has to change here, just mentioning